enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
@mattdrayer Seems like a reasonable compromise to me.

@bdero this would require another change to add a `EnterpriseContactUserSerializer` class, pulled from the user that matches this ID.

What do you think?